src/test/java/unit/CromwellConfig.java

```diff
@@ -0,0 +1,33 @@
+package unit;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import harness.TestCommand;
+import harness.baseclasses.SingleWorkspaceUnit;
+import java.io.File;
+import java.io.IOException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+/** Tests for the `terra cromwell` commands. */
+@Tag("unit")
+public class CromwellConfig extends SingleWorkspaceUnit {
+  @Test
+  @DisplayName("app-launch config affects how apps are launched")
+  void cromwellConfig() throws IOException {
+    workspaceCreator.login();
+
+    // `terra workspace set --id=$id`
+    TestCommand.runCommandExpectSuccess("workspace", "set", "--id=" + getUserFacingId());
+
```

Comment:
you might need to create a folder here.

and then at 25, cromwell generate-config --path=/path/to/the/folder/you/just/created.